openhands/runtime/impl/remote/remote_runtime.py

```diff
@@ -317,6 +320,10 @@ def vscode_url(self) -> str | None:
         else:
             return None
 
+    @property
+    def web_hosts(self) -> list[str]:
+        return self.open_ports
```

Comment:
I don't think this should be `ports`. We'll need a FQDN, since it'll be different for each runtime

Reply:
Forgot to rename these!